libraries/node-core-library/src/test/Path.test.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+/// <reference types='mocha' />
+
+import { Path } from '../Path';
+import { assert } from 'chai';
+
+describe('Path', () => {
+  describe('Test', () => {
+    it('Windows paths', () => {
+      assert.isTrue(Path.isUnder('C:\\a\\b.txt', 'C:\\a'), '1');
+      assert.isTrue(Path.isUnder('C:\\a\\b.txt', 'C:\\a\\'), '2');
+      assert.isTrue(Path.isUnder('C:\\a\\b\\c.txt', 'C:\\a'), '3');
+
+      assert.isFalse(Path.isUnder('C:\\a\\b.txt', 'C:\\b'), '4');
+      assert.isFalse(Path.isUnder('C:\\a\\b.txt', 'C:\\b\\'), '5');
+      assert.isFalse(Path.isUnder('C:\\a\\b\\c.txt', 'C:\\b'), '6');
+    });
+    it('Unix paths', () => {
+      assert.isTrue(Path.isUnder('/a/b.txt', '/a'), '1');
```

Comment:
>'/a/b.txt', [](start = 33, length = 11)

IIRC on windows these will just be converted to windows-style paths (because you aren't using path.posix)